src/UiPath.Workflow.Runtime/ProcessActivityTreeOptions.cs

```diff
@@ -7,7 +7,7 @@ namespace System.Activities;
 using Runtime;
 using Validation;
 
-internal class ProcessActivityTreeOptions
+public class ProcessActivityTreeOptions
```

Comment:
This class needs to be `public` because method `CacheRootMetadata` from [ActivityUtilities class](https://github.com/UiPath/CoreWF/blob/7d795e5a67cfd7df6dc492ed9d33f256ddfd7952/src/UiPath.Workflow.Runtime/ActivityUtilities.cs#L738) needs to be `public` in order to not break the reflection implementation of obtaining the method in [Studio.sln](https://github.com/UiPath/Studio/blob/ea50dd928156bffcac58b65ef908b72eb221fde0/Studio/UiPath.ActivityPreprocessor/WorkflowBuilder/TemplatedWorkflowBuilder.cs#L144C42-L144C59) .